lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -193,18 +193,14 @@ def flush
 
   def triggers_same?(current_trigger, desired_trigger)
     return false if current_trigger.has_key?('enabled') && !current_trigger['enabled']
```

Comment:
Is this actually a test we want to do? What if we desire the trigger to be disabled?

Reply:
Filed a ticket for this: